src/Message/CompletePurchaseRequest.php

```diff
@@ -37,9 +37,12 @@ public function getData()
 
     public function send()
     {
-        $request = $this->httpClient->get($this->endpoint);
-        $request->getQuery()->replace($this->getData());
-        $httpResponse = $request->send();
+
+        $data = $this->getData();
+        $postdata = http_build_query($data);
+        $httpRequest = $this->httpClient->post($this->getEndPoint($postdata), null, $data);
```

Comment:
This is the main fix for this PR

Reply:
So this mainly just switches from using GET to POST?

Why bother renaming the variables for request/response?

Reply:
It was a copy+paste from master. I'll rename back.

Reply:
I don't mind the rename (it's probably more consistent in modern Omnipay, I was just curious since from memory that wasn't the convention for Omnipay 1.